### PR TITLE
added missing yup in package.json

### PR DESCRIPTION
**Pull request by roy-sagar11**

### CODE REVIEW

File: app/src/hooks/useGetProjects.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import useAxios from 'axios-hooks';
+import React from 'react';
+
+function useGetProjects() {
+  const [{ data, error, loading }, refetch] = useAxios(
+    'http://localhost:4000/api/v1/project/mock/data',
```
Comment (kushal-arora-fw) [inline app/src/hooks/useGetProjects.ts:6]: endpoint hardcoded here, is it intentional? @roy-sagar11 
Can we put the endpoint in some config, maybe?

Comment (will-freshworks) [reply app/src/hooks/useGetProjects.ts:6]: Either config, or use a relative address `/api/v1/...` for local development as well (you'd need to use the react proxy I think)

**Review by will-freshworks — CHANGES_REQUESTED** [review]
Same comment as Kushal about the hardcoded URL

File: app/src/hooks/useGetProjects.ts (hunk 1 above)
Comment (fwpushan) [reply app/src/hooks/useGetProjects.ts:6]: @roy-sagar11 @matbusby-fw We can use axios `interceptors` for centralise  configuration

Comment (roy-sagar11) [reply app/src/hooks/useGetProjects.ts:6]: ohh .. that was my mistake. i was working on a different branch and stashed the changed but forgot to add this new files to git. so it didn;t get shashed and came with the fix branch. i'll open a pr with this changes later

**Pull request closed without merging**